package.json

```diff
@@ -6,19 +6,37 @@
   "scripts": {
     "dev": "vite",
     "build": "tsc && vite build",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "prepare": "husky install"
   },
   "dependencies": {
+    "commitizen": "^4.2.5",
```

Comment:
Maybe commitizen should also be moved to "devDependencies"?

Reply:
Thank you!  added to devDependencies